pkg/manager-nnf/manager.go

```diff
@@ -1416,6 +1416,10 @@ func (*StorageService) StorageServiceIdFileSystemIdExportedSharesPost(storageSer
 		}
 
 		if err := sg.serverStorage.MountFileSystem(fs.fsApi, sh.mountRoot); err != nil {
+			if deleteErr := sg.serverStorage.DeleteFileSystem(fs.fsApi); deleteErr != nil {
+				return ec.NewErrInternalServerError().WithResourceType(FileShareOdataType).WithError(deleteErr).WithCause(fmt.Sprintf("File share '%s' failed delete", sh.id))
```

Comment:
Does this tell us that we were doing the delete because the mount failed?  We want that error info from the Mount failure.

Reply:
You can tell when the error is printed by the nnf-node-manager that it was from a file share creation failure. I can edit this message to make it extra clear.